pkg/cmd/cli/cmd/deploy.go

```diff
@@ -350,7 +350,7 @@ func (o DeployOptions) cancel(config *deployapi.DeploymentConfig) error {
 				continue
 			}
 
-			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = deployapi.DeploymentCancelledAnnotationValue
+			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = unversioned.Now().Format(time.RFC3339)
```

Comment:
I also think that this breaks backwards-compatibility for `oc deploy --cancel` on older servers (because older servers expect "true" as a value in order to consider a deployment as cancelled). :/